core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -53,8 +53,6 @@ private[data] sealed abstract class WriterTInstances extends WriterTInstances0 {
   implicit def writerTIdMonad[L:Monoid]: Monad[WriterT[Id, L, ?]] =
     writerTMonadWriter[Id, L]
 
-  // The Eq[(L, V)] can be derived from an Eq[L] and Eq[V], but we are waiting
-  // on an algebra release that includes https://github.com/non/algebra/pull/82
   implicit def writerTIdEq[L, V](implicit E: Eq[(L, V)]): Eq[WriterT[Id, L, V]] =
```

Comment:
Should we change this to `writerTIdEq[L : Eq, V : Eq]` instead of requiring the instance on the tuple